src/plugin.ts

```diff
@@ -30,3 +33,16 @@ export const XkcdComicCard =
             },
         }),
     );
+
+export const XkcdComicHomePageComponent = homePlugin.provide(
+    createCardExtension<{ defaultCategory?: 'any' | 'humor' }>({
+        name: 'XkcdComicCard',
+        title: 'xkcd',
+        components: () => import('./components/XkcdComicCardHomePage'),
+        layout: {
+            height: { minRows: 1 },
+            width: { minColumns: 3 },
```

Comment:
You could also add settings here so that user can change for example to show some static xkcd strip or hide the navigation buttons. See the homepage docs for info; it's basically just a schema for JSON Schema Form.